sections/05_treat.md

```diff
@@ -162,45 +162,193 @@ feature selection and construction.*
+between random forests and a deep neural were negligible, especially in the
+classification setting.  Overfitting is still a problem in larger chemical
+screening datasets with tens or hundreds of thousands of compounds because the
+number of active compounds can be very small, on the order of 0.1% or 1% of all
+tested chemicals `TODO: verify those estimates`. This is consistent with the
+strong performance of low-parameter neural networks that emphasize
+compound-compound similarity, such as influence-relevance voter,
+[@tag:Swamidass2009_irv  @tag:Lusci2015_irv] instead of predicting compound
+activity directly from chemical features. `TODO: include recent DeepChem IRV
+benchmarks?`
+
+Much of the recent excitement in this domain has come from what could be
+considered a creative experimentation phase, in which deep learning has offered
+novel possibilities for feature representation and modeling of chemical
+compounds.  A molecular graph, where atoms are nodes and bonds are edges, is a
+natural way to represent a chemical structure.  Traditional machine learning
+approaches relied on preprocessing the graph into a feature vector, such as a
+fixed-width bit vector fingerprint [@tag:Rogers2010_fingerprints] in which each
+bit represents the presence of absence of a particular chemical substructure in
+the molecular graph. `TODO: this is a massive oversimplification because there
+can be collisions, but I don't know how much detail to get into` Modern neural
+networks can operate directly on the molecular graph as input.  Duvenaud et al
+[@tag:Duvenaud2015_graph_conv] generalized standard circular fingerprints by
+substituting discrete operations in the fingerprinting algorithm with operations
+in a neural network, producing a real-valued feature vector instead of a bit
+vector.  Other approaches offer trainable networks that can in theory learn
+chemical feature representations that are optimized for a particular prediction
+task.   Lusci et al adapted recursive neural networks for directed acyclic
+graphs for undirected molecular graphs by creating an ensemble of directed
+graphs in which one atom is selected as the root node [@tag:Lusci2013_rnn].  A
+single feature vector is obtained by summing over all feature vectors for all
+directed graphs in the ensemble.  Graph convolutions on undirected molecular
+graphs have eliminated the need to enumerate artificial directed graphs,
+learning feature vectors for atoms that are a function of the properties of
+neighboring atoms and local regions on the molecular graph
+[@tag:Kearnes2016_graph_conv @tag:AltaeTran2016_one_shot]. `TODO: add more
+details on how graph convolutions work?`
+
+Advances in chemical representation learning have not been limited to
+graph-based neural networks.  The simplified molecular-input line-entry system
+(SMILES) is a standard way to transform a chemical into string-based
+representation.  A SMILES-to-SMILES autoencoder learns a continuous latent
```

Comment:
Good point @mrwns.  I included the autoencoder here with the supervised methods because I want to focus on learned feature representations.  But that is an awkward fit.  I have a TODO to add GANs and other methods and will note that I need to revisit the autoencoder's placement as well.